system-parachains/bridge-hubs/bridge-hub-polkadot/src/xcm_config.rs

```diff
@@ -163,7 +169,11 @@ impl Contains<RuntimeCall> for SafeCallFilter {
 			RuntimeCall::System(frame_system::Call::set_storage { items })
 				if items.iter().all(|(k, _)| {
 					k.eq(&DeliveryRewardInBalance::key()) ||
-						k.eq(&RequiredStakeForStakeAndSlash::key())
+						k.eq(&RequiredStakeForStakeAndSlash::key()) ||
+						k.eq(&EthereumGatewayAddress::key()) ||
+						// Allow resetting of Ethereum nonces in Rococo only.
```

Comment:
Rococo? Do we need it here?

Reply:
Good catch, removed in https://github.com/polkadot-fellows/runtimes/pull/130/commits/575155243e4f0d4b92af7907f231ac48baa901fb.